server

```diff
@@ -114,14 +52,23 @@ check_product() {
   esac
 }
 
-is_rc_version() {
-    if [[ $SCYLLA_VERSION == *.rc* ]]; then
-      # replace the 2nd '.' with '~' for RC versions, ex. 4.6.rc0 -> 4.6~rc0
-      SCYLLA_VERSION=$(echo $SCYLLA_VERSION | sed 's/\(.*\)\.)*/\1~/')
-    elif [[ $SCYLLA_VERSION == *-rc* ]]; then
-      # replace '-' with '~' for RC versions, ex. 5.1.0-rc1 -> 5.1.0~rc1
-      SCYLLA_VERSION=$(echo $SCYLLA_VERSION | sed 's/-/~/')
+check_scylla_version() {
+  if [[ ! -z "$SCYLLA_VERSION" ]] && [[ $SCYLLA_VERSION != *"nightly"* ]]; then
```

Comment:
Replace `! -z` with `-n`

Reply:
btw, I'm not familiar with the whole script-logic but maybe the not-null check is not required and `$SCYLLA_VERSION != *"nightly"*` is enough

Reply:
changed to `-n`. we need to check of null param cause we have an option of `./server` installation and `query_default_version` funtion comes only after